bundles/org.openhab.persistence.jdbc/src/main/java/org/openhab/persistence/jdbc/db/JdbcBaseDAO.java

```diff
@@ -342,10 +342,14 @@ public void doStoreItemValue(Item item, ItemVO vo) {
         String sql = histItemFilterQueryProvider(filter, numberDecimalcount, table, name, timeZone);
         logger.debug("JDBC::doGetHistItemFilterQuery sql={}", sql);
         List<Object[]> m = Yank.queryObjectArrays(sql, null);
+        if (m == null) {
```

Comment:
Is this a bug in Yank? Seems it would be more appropriate if it would have thrown `YankSQLException`, as this seems to be an error situation? Do we have correct error handling in place for this exception - if so, maybe throw instead of returning empty list? Sorry, no time for digging deeper right now, so for now just asking questions. :)

Reply:
`YankSQLException` have been introduced in version 3.3.0. We are using version 3.2.0 which handles all exceptions internally by logging an error. Thus the return value of this method is `null`.